frontend/containers/DashboardContainer.vue

```diff
@@ -25,6 +30,13 @@ export default Vue.extend({
                 this.$store.dispatch('preferences/setExplanationsAction', newIsExplanationsShown);
             },
         },
+        callees(): CalleeAddresses {
+            const pageNetwork = this.$store?.getters['network/getPageNetwork'];
+            if (pageNetwork === 'stub') {
+                return {};
+            }
```

Comment:
As `stub` string is configurable and there can be more networks, let's instead use `getMakerNetwork` and check that it's not empty here